### PR TITLE
One True Mode ™️

### PR DESCRIPTION
**Pull request by samouri**
**summary**
Removes `isProd` in favor of _only_ having `isTest`

cc @ampproject/wg-performance

### CODE REVIEW

**Comment by kristoferbaxter** [comment]
image

Love it.

File: test/unit/3p/test-3p-frame.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ describes.realWin('3p-frame', {amp: true}, (env) => {
 
   function mockMode(options) {
     env.sandbox.stub(window.parent, '__AMP_MODE').value(options);
-    env.sandbox.stub(mode, 'isProd').returns(!!options.isProd);
+    env.sandbox.stub(mode, 'isTest').returns(!!options.test);
```
Comment (rcebulko) [inline test/unit/3p/test-3p-frame.js:32]: We should provide this as a helper util

**Review by rcebulko — APPROVED** [review]
It's been a long journey to get here 🥳 🎉

**Review by jridgewell — APPROVED** [review]

**Comment by samouri** [comment]
With the revert of the `localDev` changes, I need to descope this to "Two true modes" for now. Getting rid of localDev may be a longer tail project. Will update this PR either tomorrow or early next week.

**Pull request closed without merging**